torch/distributed/pipelining/_backward.py

```diff
@@ -38,7 +37,7 @@ def _get_grad_fn_or_grad_acc(t: torch.Tensor) -> Union[Node, None]:
 
 
 def reverse_closure(
-    roots: List[Node], target_nodes: Set[Node]
+    roots: List[Node], target_nodes: Set[Node], reverse_edges_dict
```

Comment:
nit: add doc to the `reverse_edges_dict` argument.

Reply:
will do!